code/style.css

```diff
@@ -2,6 +2,164 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+html,
+body {
+    height: 100%;
+    background-color: #c9c5c2;
 }
+
+
+/* The accordion when it's expanded*/
+
+.active+.answer {
```

Comment:
This is completely valid CSS, it would look far more aesthetic written like this : `.active + .answer`